packages/search-ui-workplace-search-connector/src/__tests__/WorkplaceSearchAPIConnector.test.ts

```diff
@@ -98,7 +99,7 @@ describe("WorkplaceSearchAPIConnector", () => {
         beforeAutocompleteSuggestionsCall?: SuggestionsQueryHook;
       } = {}
     ) {
-      if (!state.searchTerm) state.searchTerm = "searchTerm";
+      if (!state || !!state.searchTerm) state.searchTerm = "searchTerm";
```

Comment:
Should it be `!state.searchTerm`? (one `!`)

Reply:
thanks for spot! Should in fact be removed as got a default object for state in the arguments and this line was never used.